### DISCUSSION TITLE
Help me with time-lapse photography

### DISCUSSION THREAD

**Discussion by DubeAlba** [gopro/labs#236] [Q&A]
Hello. I am using the GoPro 9 to carry out my time-lapse photography project. I use the start command and shoot in preset mode before getting power from the charger. The problem I'm facing is that with this command, I always have to leave the battery inside the camera, which causes the battery to swell. I'm wondering if there is a command that can allow me to shoot directly from the USB power source without needing a battery inside. Thank you.

**Comment by dnewman-gpsw** [comment]
Battery is not needed if you have continuous USB power.

**Comment by DubeAlba** [comment]
I don't have a continuous source of electricity where I am. The power will be turned off during lunch breaks and after working hours. The command I'm using to start the time-lapse when the USB power is supplied and stop when the power is cut off is very suitable for me. But I have a problem because I have to leave the battery in the camera. My goal is for the camera to turn on and capture from 7am to 5pm, but I'm experiencing power outages around noon. Can you advise me on any other solutions? Thank you.
![IMG_20230215_094725](https://user-images.githubusercontent.com/125379576/218915733-511a9b95-1e4c-44b9-bf55-3da4359b753b.jpg)

**Reply by dnewman-gpsw** [reply]
What you would normally use instead is a boot command. Unfortunately HERO9 is missing the needed WAKE=2 support. The older WAKE=1 is on HERO9, can't be used.  If you have H8, H10 or H11 you can run with a battery using this
![image](https://user-images.githubusercontent.com/10856084/218945722-e34bfa5e-09ae-4b76-84e5-85976ef88421.png)
